src/harness/reference_models/move_list_calc/move_list.py

```diff
@@ -0,0 +1,582 @@
+                        'latitude', 'longitude', 'lowFrequency', 'highFrequency'
+        h_inc_ant:      reference incumbent antenna height (in meters)
+        num_iteration:  a number of Monte Carlo iterations
+        dpa_type:       DPA type, i.e., 'co-channel offshore',
+                        'co-channel inland', or 'out-of-band inland'
+
+    Returns:
+        interference: 	interference contribution, a tuple with named fields
+                        'grantIndex', 'medianInterference', 'randomInterference'
+                        (K random interference contributions of the grant to
+                        protection constraint c), and 'bearing_c_cbsd'
+                        (bearing from c to CBSD grant location).
+    """
+
+    # Get CBSD grant information
+    grant_index = cbsd_grant.grantIndex
+    lat_cbsd = cbsd_grant.latitude
+    lon_cbsd = cbsd_grant.longitude
+    h_cbsd = cbsd_grant.height
+    indoor_cbsd = cbsd_grant.indoorDeployment
+    ant_beamwidth_cbsd = cbsd_grant.antennaBeamwidth
+    ant_gain_cbsd = cbsd_grant.antennaGain
+    ant_az_cbsd = cbsd_grant.antennaAzimuth
+    maxEirpPerMHz_cbsd = cbsd_grant.maxEirp
+    low_freq_cbsd = cbsd_grant.lowFrequency
+    high_freq_cbsd = cbsd_grant.highFrequency
+
+    # Get protection constraint information (protection point and frequency range)
+    lat_c = constraint.latitude
+    lon_c = constraint.longitude
+    low_freq_c = constraint.lowFrequency
+    high_freq_c = constraint.highFrequency
+
+    # Compute median and K random realizations of path loss/interference contribution
+    # based on ITM model as defined in [R2-SGN-03] (in dB)
+    reliabilities = np.random.uniform(0.001, 0.999, num_iteration)  # get K random
+                # reliability values from an uniform distribution over [0.001,0.999)
+    reliabilities = np.append(reliabilities, [0.5])  # add 0.5 (for median loss) as
+                                            # a last value to reliabilities array
+    results = wf_itm.CalcItmPropagationLoss(lat_cbsd, lon_cbsd, h_cbsd,
+                                            lat_c, lon_c, h_inc_ant,
+                                            indoor_cbsd,
+                                            reliability=reliabilities,
+                                            freq_mhz=FREQ_PROP_MODEL)
+    path_loss = np.array(results.db_loss)
+
+    # Compute EIRP of CBSD grant inside the frequency range of protection constraint
+    if dpa_type != 'out-of-band inland': # For co-channel offshore/inland DPAs
+
+        # CBSD co-channel bandwidth overlapping with
+        # frequency range of protection constraint
+        co_channel_bw = min(high_freq_cbsd, high_freq_c) - max(low_freq_cbsd, low_freq_c)
+
+        # Overlapping bandwidth between the first 10 MHz of CBSD grant adjacent-channel and
+        # the frequency range of protection constraint
+        adj_channel_bw = min(10000000.0, high_freq_c - low_freq_c - co_channel_bw)
```

Comment:
Can you please share with me the spec describing the adjacent channel inclusion? I coulkd not find it among the various documentation.